src/character.py

```diff
@@ -12,37 +14,55 @@ class DinoSprite(pygame.sprite.Sprite):
     """Initialize DinoSprite"""
     # pylint: disable=C0303
     # pylint: disable=R0903
+
+    # jump speed
+    jump_speed = 10
```

Comment:
In future, I think it'd be nice if we made a separate file with all these constants